web/src/i18n.ts

```diff
@@ -0,0 +1,8 @@
+
+// IMPORTANT: This file adds comments recognized by the react-i18next-parser so that
+// labels declared in console-extensions.json are added to the message catalog.
+
+// TODO: We should make the custom i18next JSON lexer available to plugins so
```

Comment:
is it a TODO for us, or it was copy/pasted from console repo ?

Reply:
Copy/paste from console, sorry